examples/grib_info_download.py

```diff
@@ -5,11 +5,30 @@
 import pygrib
 import bokeh.models.sources
 import io
+import logging
+
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+logger.setLevel(logging.ERROR)
+
+# logger = logging.getLogger(__name__)
```

Comment:
Please remove the commented out code as well as the _file_handler that you're not really using...